CRM/Contact/Form/Edit/TagsAndGroups.php

```diff
@@ -135,7 +135,7 @@ public static function buildQuickForm(
       $tags = CRM_Core_BAO_Tag::getColorTags('civicrm_contact');
 
       if (!empty($tags)) {
-        $form->add('select2', 'tag', ts('Tag(s)'), $tags, FALSE, ['class' => 'huge', 'placeholder' => ts('- select -'), 'multiple' => TRUE]);
+        $form->add('select2', $fieldName, $tagName, $tags, FALSE, ['class' => 'huge', 'placeholder' => ts('- select -'), 'multiple' => TRUE]);
```

Comment:
This makes the tags field disappear on the contact edit form (in the tags and groups accordion at the bottom). Suggest maybe:
`$form->add('select2', $fieldName ?? 'tag', $tagName, ...`

Reply:
But there's still a little more needed even, because no default value gets set for the profile field so it appears blank when using batch update on a contact that already has a tag.

Reply:
Oh I see you already noted that in the ticket. Sorry didn't read first.

Reply:
> This makes the tags field disappear on the contact edit form (in the tags and groups accordion at the bottom). Suggest maybe:
> `$form->add('select2', $fieldName ?? 'tag', $tagName, ...`

ah, thanks for suggestion